### PR TITLE
Add event entities to Folder watcher

### PR DESCRIPTION
**Pull request by gjohansson-ST**
## Proposed change

Adds event entities to Folder watcher

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/116526
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/folder_watcher.markdown [hunk 1]
```diff
@@ -28,6 +30,11 @@ Configured folders must be added to [allowlist_external_dirs](/integrations/home
 Pattern matching using [fnmatch](https://docs.python.org/3.6/library/fnmatch.html) can be used to limit filesystem monitoring to only files which match the configured patterns.
 As example to monitor specific file, as example YAML and text-files add `*.yaml` and `*.txt`.
 
+## Entities
+
+This integration will besides the events also create `event` entities for the respective events that can happen.
+These entities can be used for automations instead of using the respective event.
```
Comment (frenck) [inline source/_integrations/folder_watcher.markdown]: I think we should make this the main focus for the documentation (and not the old style events, which we can just fully remove from the docs IMHO).

**Review by frenck — DISMISSED** [review]
Small comment

File: source/_integrations/folder_watcher.markdown (hunk 1 above)
Comment (gjohansson-ST) [reply source/_integrations/folder_watcher.markdown]: Modified accordingly. Hope that fits so it's understandable.

File: source/_integrations/folder_watcher.markdown [hunk 2]
```diff
@@ -7,11 +7,15 @@ ha_iot_class: Local Polling
...
+ha_platforms:
+  - event
 ha_domain: folder_watcher
 ha_integration_type: integration
 ---
 
-The **Folder watcher** {% term integration %} adds [Watchdog](https://pythonhosted.org/watchdog/) file system monitoring, publishing events on the Home Assistant bus on the creation/deletion/modification of files within configured folders. The monitored `event_type` are:
+The **Folder watcher** {% term integration %} adds [Watchdog](https://pythonhosted.org/watchdog/) file system monitoring.
+
+It creates Event entities for these monitored event types:
```
Comment (c0ffeeca7) [inline source/_integrations/folder_watcher.markdown]:
```suggestion
It creates event entities for these monitored event types:
```

File: source/_integrations/folder_watcher.markdown [hunk 3]
```diff
@@ -41,7 +45,7 @@ When the `event_type` is `moved`, the file details are for the source file and d
 - `dest_file`: The name of the moved file (e.g. "world.txt")
 - `dest_folder`: The folder moved path (e.g. "/hello")
 
-Automations can be triggered on filesystem event data using a template. The following automation will send a notification with the name and folder of new files added to that folder:
+Automations can be triggered on filesystem events data using a template. The following automation will send a notification with the name and folder of new files added to that folder:
```
Comment (c0ffeeca7) [inline source/_integrations/folder_watcher.markdown]:
```suggestion
Automations can be triggered on file system events data using a template. The following automation will send a notification with the name and folder of new files added to that folder:
```

**Review by c0ffeeca7 — APPROVED** [review]
Thank you, @gjohansson-ST :+1:

**Pull request merged**